api/controllers/AuthController.js

```diff
@@ -86,6 +86,12 @@ module.exports = {
       inviteToken : req.param('inviteToken')
     };
 
+    // checking password for white spaces and tabs
```

Comment:
`\s` is the regex standard for whitespace. Not sure all browsers would treat `/ /` correctly